components/MintButtonV3.vue

```diff
@@ -107,6 +111,21 @@ export default {
 					'any'
 				)
 		)
+		const isMobile = computed(() => {
```

Comment:
let's either use a library or make this a hook or move to the utils.js

Reply:
made this computed value as a hook